xamarin/FHXamarinAndroidSDK/Services/Device/DeviceService.cs

```diff
@@ -96,6 +98,56 @@ public string GetPackageDir()
 		{
 			return "./";
 		}
+
+		public PushConfig ReadPushConfig ()
+		{
+			Properties appProps = new Properties();
+			Stream input = null;
+			ILogService logger = ServiceFinder.Resolve<ILogService>();
+			try {
+				input = Application.Context.Assets.Open(APP_PROP_FILE);
+				appProps.Load(input);
+
+				string pushServerUrl = appProps.GetProperty("PUSH_SERVER_URL");
```

Comment:
@matzew fixed that in my PR to this PR https://github.com/secondsun/fh-dotnet-sdk/pull/1